packages/react-router-redux/modules/ConnectedRouter.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component, PropTypes } from 'react'
+import { Router, Route } from 'react-router'
+
+import { LOCATION_CHANGE } from './reducer'
+
+class ConnectedRouter extends Component {
+  static propTypes = {
+    store: PropTypes.object,
+    history: PropTypes.object,
```

Comment:
I wasn't sure if we wanted to provide a reasonable default here (create a browser history if none provided) or set up prefab `<Connected(Browser/Hash/Memory)Router>` exports.